src/indicators/moving_standard_deviation.rs

```diff
@@ -0,0 +1,178 @@
+use std::fmt;
+
+use crate::errors::*;
+use crate::{Close, Next, Reset};
+
+/// Moving Standard Deviation (MSD).
+///
+/// # Formula
+///
+/// ![MSD](https://wikimedia.org/api/rest_v1/media/math/render/svg/9f2b2061a0bda785c7d246d21ec20a17c79185e0)
+///
+/// Where:
+///
+/// * _MSD<sub>t</sub>_ - value of standard deviation at a point of time _t_
+/// * _n_ - number of periods (length)
+/// * _p<sub>t</sub>_ - input value at a point of time _t_
+///
+/// # Parameters
+///
+/// * _n_ - number of periods (integer greater than 0)
+///
+/// # Exampile
+///
+/// ```
+/// use ta::indicators::MovingStandardDeviation;
+/// use ta::Next;
+///
+/// let mut msd = MovingStandardDeviation::new(3).unwrap();
+/// assert_eq!(msd.next(1.0), 1.4142135623730951);
+/// assert_eq!(msd.next(2.0), 1.1726039399558574);
+/// assert_eq!(msd.next(3.0), 0.816496580927726);
+/// ```
+///
+/// # Links
+///
+/// * [Standard Deviation, Wikipedia](https://en.wikipedia.org/wiki/Standard_deviation)
+///
+#[derive(Debug, Clone)]
+pub struct MovingStandardDeviation {
+    n: u32,
+    index: usize,
+    count: u32,
+    sum: f64,
+    vec: Vec<f64>,
```

Comment:
`vec` could you please use another name (e.g. `values` or `prices`).
`vec` could be a little confusing because of the macro `vec!`.